src/spath.c

```diff
@@ -1455,3 +1461,36 @@ int spath_is_writeable(const spath* file)
   return rc;
 }
 #endif
+
+//#ifndef HIDE_TV
+/*
+=========================================
+Pretty print for TotalView debug window
+=========================================
+*/
+
+/* This enables a nicer display when diving on a path variable
+ * under the TotalView debugger.  It requires TV 8.8 or later. */
+
+//#include "tv_data_display.h"
+
+//static int TV_ttf_display_type(const spath* path)
+//{
+//  if (path == NULL) {
+    /* empty path, nothing to display here */
+//    return TV_ttf_format_ok;
+//  }
+
+//  if (spath_is_null(path)) {
+    /* empty path, nothing to display here */
+//    return TV_ttf_format_ok;
+//  }
+
+  /* print path in string form */
+//  char* str = spath_strdup(path);
+//  TV_ttf_add_row("path", TV_ttf_type_ascii_string, str);
+//  spath_free(&str);
+
+//  return TV_ttf_format_ok;
+//}
+//#endif /* HIDE_TV */
```

Comment:
Cleaner way to comment a bunch of lines
```suggestion
#if 0
#ifndef HIDE_TV
/*
=========================================
Pretty print for TotalView debug window
=========================================
*/

/* This enables a nicer display when diving on a path variable
 * under the TotalView debugger.  It requires TV 8.8 or later. */

#include "tv_data_display.h"

static int TV_ttf_display_type(const spath* path)
{
  if (path == NULL) {
    /* empty path, nothing to display here */
    return TV_ttf_format_ok;
  }

  if (spath_is_null(path)) {
    /* empty path, nothing to display here */
    return TV_ttf_format_ok;
  }

  /* print path in string form */
  char* str = spath_strdup(path);
  TV_ttf_add_row("path", TV_ttf_type_ascii_string, str);
  spath_free(&str);

  return TV_ttf_format_ok;
}
#endif /* HIDE_TV */
#endif
```